test/test_contra_env_setup.py

```diff
@@ -37,73 +52,68 @@ def setUp(self):
         self.vm.args.extend(['-cpu', 'host'])
         self.vm.args.extend(['-accel', 'kvm'])
 
-        # Headless
+        # headless
         self.vm.args.extend(['-display', 'none', '-vga', 'none'])
 
-        # Run!
+        # run!
         self.vm.launch()
 
-        # Wait VM to be UP
+        # wait VM to be UP
         with GetConsole(self.vm) as console:
             console.sendline('sudo resize2fs /dev/sda1')
```

Comment:
this command could be part of the first ansible AdHoc command. You'd increase the debug-ability.

Reply:
do you mean the first command in the test() function?

Reply:
Yes. In that case, you will not need `sudo`